lib/docker_sync/sync_strategy/unison.rb

```diff
@@ -61,10 +61,13 @@ def sync_options
         args.push(@options['src'])
         args.push('-auto')
         args.push('-batch')
+        args.push('-owner') if @options['sync_userid'] == 'from_host'
+        args.push('-group') if @options['sync_groupid'] == 'from_host'
+        args.push('-numericids') if @options['sync_userid'] == 'from_host' || @options['sync_groupid'] == 'from_host'
         args.push(@options['sync_args']) if @options.key?('sync_args')
         args.push("socket://#{@options['sync_host_ip']}:#{@options['sync_host_port']}/")
         args.push('-debug verbose') if @options['verbose']
-        if @options.key?('sync_user') || @options.key?('sync_group') || @options.key?('sync_groupid') || @options.key?('sync_userid')
+        if @options.key?('sync_user') || @options.key?('sync_group') || @options.key?('sync_groupid') && @options['sync_groupid'] != 'from_host' || @options.key?('sync_userid') && @options['sync_userid'] != 'from_host'
```

Comment:
that looks pretty good. Still ,you need to add this option to example/docker-sync.yml and document it. We should then either offer the same for rsync or state that this is unison only and catch this in rsync as a exception the same way i did this for excludes / users in unison before